monitoring/monitoring.go

```diff
@@ -149,6 +149,7 @@ var (
 	systemDiskUsage     = disk.Usage
 	getUserFSQuotaUsage = fs.GetUserFSQuotaUsage
 	getProcesses        = getProcessesList
+	numCPU              = runtime.NumCPU()
```

Comment:
you could squash test commit whith implementation

Reply:
+1, otherwise change commit message: update tests with removed data channel or something.